_layouts/post.html

```diff
@@ -37,8 +37,8 @@
           </div>
 
           <div class="article-share">
-            {% assign page_title = page.title  | truncate '110' %}
-            <a href="" title="Share on Twitter" onclick="window.open('https://twitter.com/home?status={{ page_title }} - {{ page.url | absolute_url }} {% if site.social.twitter %}by @{{ site.social.twitter }}{% endif %}', 'newwindow', 'width=500, height=225'); return false;" data-turbolinks="false">
+            {% assign page_title = page.title | truncate '110' %}
+            <a href="" title="Share on Twitter" onclick="window.open('https://twitter.com/home?status={{ page_title | cgi_escape }} - {{ page.url | absolute_url }} {% if site.social.twitter %}by @{{ site.social.twitter }}{% endif %}', 'newwindow', 'width=500, height=225'); return false;" data-turbolinks="false">
```

Comment:
Please add cgi_escape to the page.title above this line so it is added in every link.